examples/simple_plugin/go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/cloudquery/plugin-sdk/examples/simple_plugin
```

Comment:
These changes can be reverted once https://github.com/cloudquery/plugin-pb-go/pull/401 is merged and released